docs/INSTALLATION.md

```diff
@@ -1,65 +1,90 @@
 # Installation
 
-## Installing Stack
 
-If you don't have a recent Stack version installed in your system, follow the
-[installation instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/)
-for your operating system. After completing the installation, test that you
-have installed it correctly by running this command in a terminal:
+## Get GHCup
 
-```bash
-stack --version
-```
+The recommended way of installing Haskell tools is through [GHCup][ghcup].
 
-If it outputs a Stack version equal to or greater than 1.1.2, then you have it
-installed correctly.
+With GHCup you can manage your installed compilers, project/package managers, and language server.
 
-## Installing GHC
 
-Stack will automatically download and install GHC (the Glasgow Haskell
-compiler) when it is required. You can start the installation and test
-by running the following command in a terminal:
+## Install Stack
 
-```bash
-stack ghc
-```
+For doing Exercism exercises you only need Stack.
+
+To install Stack, simply run the GHCup installation command or script and follow the defaults but specify that you would like to have Stack installed as well.
```

Comment:
```suggestion
To install Stack, run the GHCup installation command or script and follow the defaults but specify that you would like to have Stack installed as well.
```

This is a complex sentence and "simply" is not implied, as we have to make decisions here.  Often "simply" is insulting to see when and if we have a problem.  We can avoid the "insult" by removing this one word.  Otherwise it does encourage one to believe that they have not understood the "simple" and are somehow the problem.